src/pages/all-about-carbon/the-team.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: The team
+description:
+  Carbon is not just one team. To sustain productive contribution and maintain a
+  clear direction for Carbon as a whole, various groups works together to
+  provide direction, governance, and support.
+---
+
+<PageDescription>
+
+Carbon is not just one team. To sustain productive contribution and maintain a
+clear direction for Carbon as a whole, various groups works together to provide
+direction, governance, and support.
+
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Overview</AnchorLink>
+  <AnchorLink>The Carbon team</AnchorLink>
+  <AnchorLink>Steering committee</AnchorLink>
+  <AnchorLink>Working groups</AnchorLink>
+  <AnchorLink>Board of directors</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Carbon is funded and built by IBM. That means we build for the company’s
+business needs. Carbon is also an open source community-driven project. It is
+available for anyone to use and everyone is encouraged to contribute.
+
+To sustain productive contribution and maintain a clear direction for Carbon as
+a whole, there are various groups within IBM that provide governance and
+support.
+
+## The Carbon team
```

Comment:
What about `Core team` instead of `The Carbon team`? 1) It's what we call ourselves today. 2) It's awkward to say "Carbon is not just one team" and then "The Carbon team" to be one of the teams. If "The team", is everybody, I'd expect "The Carbon team" to also be everybody, where "Core team" is a subteam.